src/Explorer/Controls/NotebookViewer/NotebookViewerComponent.tsx

```diff
@@ -140,6 +138,7 @@ export class NotebookViewerComponent extends React.Component<
               onFavoriteClick={this.favoriteItem}
               onUnfavoriteClick={this.unfavoriteItem}
               onDownloadClick={this.downloadItem}
+              onReportAbuseClick={this.state.galleryItem.isSample ? undefined : this.reportAbuse}
```

Comment:
>onReportAbuseClick [](start = 14, length = 18)

We should not even expose Report abuse button for the sample notebooks.

Reply:
This is doing exactly that. If it's a sample notebook we don't set onReportAbuseClick which hides the option altogether.